docs/sdk/serverless-signing/combining-signatures.md

```diff
@@ -98,6 +104,6 @@ const res = await client.executeJs({
 console.log("result from singing in a lit action", res);
 ```
 
-## Using Signatures from Within A Lit Action
+## Complete Example: Signing Blockchain Transactions
 
-For an example of how you may use the signature from `signAndCombineEcdsa` from within the Lit Action see [here](./run-once.md).
+You can find a full example using the `signAndCombineEcdsa` function to sign and send a blockchain transaction on the Sepolia testnet [here](https://github.com/LIT-Protocol/developer-guides-code/tree/wyatt/sign-and-combine/sign-and-combine-ecdsa).
```

Comment:
I think @awisniew207 is changing this to submit a tx on Yellowstone instead of Sepolia